app/views/content/teaching-internship-providers.md

```diff
@@ -38,13 +38,13 @@ keywords:
       <span class="icon icon-warning" aria-hidden="true"></span>
     </div>
     <div>
-      <h2 class="heading-m">Applications now closed</h2>
+      <h2 class="heading-m">Apply from January 2023</h2>
       
-      <p>Applications for the Teaching Internships Programme in 2022 are now closed. Applications will reopen by early 2023.</p>
+      <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available on this page, once applications open.</p>
```

Comment:
@philspray I'm not sure we should talk about applying to the 'Teaching Internships Programme' - you don't apply to the programme but to individual schools offering internships. I also think 'programme' is very internal language?

Could we say something like 'You will be able to apply for a teaching internship from January 2023'?